cmd/snap/cmd_info.go

```diff
@@ -417,6 +417,25 @@ func (iw *infoWriter) printDescr() {
 	printDescr(iw, iw.theSnap.Description, iw.termWidth)
 }
 
+func (iw *infoWriter) maybePrintCategories() {
+	if !iw.verbose {
+		return
+	}
+	if len(iw.theSnap.Categories) == 0 {
+		return
+	}
```

Comment:
nitpick: I think this would still be readable in a single `if` but slightly more concise. Not a big deal so feel free to ignore if it was intentionally left separate

```suggestion
	if !iw.verbose || len(iw.theSnap.Categories) == 0 {
		return
	}
```